LayoutTests/http/wpt/mediarecorder/MediaRecorder-audio-bitrate.html

```diff
@@ -9,7 +9,7 @@
 <script>
     function record(stream, bitRate, mimeType)
     {
-        const recorder = new MediaRecorder(stream, { mimetype: mimeType, audioBitsPerSecond : bitRate });
+        const recorder = new MediaRecorder(stream, { mimeType, audioBitsPerSecond : bitRate });
```

Comment:
I don't understand this change. This function is passed a mime type string so your new test will pass something like

    { 'audio/mp4; codecs=opus', audioBitsPerSecond: 128000 }

to the `MediaRecorder` constructor, but the `options` parameter to the `MediaRecorder` constructor is a dictionary so it should pass this

    { mimetype: 'audio/mp4; codecs=opus', audioBitsPerSecond: 128000 }

Reply:
the option to be given is `mimeType` ; there was a typo. It was written `mimetype`. If you pass in the dictionary a parameter that has the same name as the key, it's redundant.
e.g. this is equivalent to: 
`const recorder = new MediaRecorder(stream, { mimeType: mimeType, audioBitsPerSecond : bitRate });`

Reply:
I've made it more explicit with `mimeType: mimeType`